change/@itwin-property-grid-react-fa8f64ac-f2f6-4472-9943-7b848897afa4.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "resolve cves",
```

These messages appear in consumer-facing changelog files - they should have a meaning to consumers and be properly formatted.

```suggestion
  "comment": "Bump dependencies.",
```